storage/uri_test.go

```diff
@@ -52,6 +52,11 @@ func TestURIPath(t *testing.T) {
 	u, err = storage.ParseURI(s)
 	assert.Nil(t, err)
 	assert.Equal(t, "example:animal:ferret:nose", u.Path())
+
+	s = "file:///C:/over/there"
+	u, err = storage.ParseURI(s)
+	assert.Nil(t, err)
+	assert.Equal(t, "/C:/over/there", u.Path())
```

Comment:
Returning a path like this for Windows is wrong. The logic needs to be updated in the `Path` function to strip out the extra slash for file:// urls on Windows.

Reply:
True, I added the logic accordingly. There are more and more `if runtime.GOOS == "windows"` and even more will come when UNC will be supported as well but I don't see a way around this.

Reply:
Hmm, a question I guess (and maybe this is one for @andydotxyz) - do we want to use runtime.GOOS as the trigger or do we want to handle "windows-looking" file URLs the same (ie correctly) regardless of what OS we're running on?

Reply:
If this is discussed, the handling of UNC paths on Windows should also be discussed.
Simple example:
`file://server/path` in Windows almost always means that any file operation then should be done on `\\server\path`, which is the UNC path build out of this URI. All `os` and `io` methods on Windows can handle the UNC path correctly but would fail with `server/path`.